test/gtest/uct/test_atomic_key_reg_rdma_mem_type.cc

```diff
@@ -8,6 +8,12 @@
 
 class uct_atomic_key_reg_rdma_mem_type : public uct_amo_test {
 protected:
+    void init() override
+    {
+        uct_amo_test::init();
+        modify_config("DM_COUNT", 0);
```

Comment:
do we create entities in uct_amo_test::init? if yes modify the config before init()